cli/tests/test_immutable_commits.rs

```diff
@@ -521,4 +521,20 @@ fn test_rewrite_immutable_commands() {
     [EOF]
     [exit status: 1]
     "##);
+    // sign
+    test_env.add_config(r#"signing.backend = "test""#);
+    let output = test_env.run_jj_in(&repo_path, &["sign", "-r=main"]);
```

Comment:
nit: use `--config=signing.backend=test` instead.

Reply:
That is much nicer, thanks!